Referenced by issue #41971, Python call bmesh.ops.weld_verts allways raise an error
